src/deploy/AllocatorDeploy.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-FileCopyrightText: © 2023 Dai Foundation <www.daifoundation.org>
```

Comment:
Shouldn't we have the `deploy` directory outside of `src`? Not sure which is the default structure now that `test` is outside.

Reply:
It was different in `dss-flappers` and `dss-direct-deposit` but you have a good point about the `test` dir.
Changed.